src/context/AppProvider/AppContext.tsx

```diff
@@ -246,6 +250,18 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     // and ensure the queryFn runs resulting in dispatches occuring to update client state
     const options = { forceRefetch: true }
 
+    // Sneaky hack to fetch cosmos SDK default opportunities for wallets that don't support Cosmos SDK
+    // We only store the validator data for these and don't actually store them in portfolio.accounts.byId[accountSpecifier].stakingDataByValidatorId
+    // Since the accountSpecifier is a zero-like address (generated and private keys burned) and isn't actually in state
+    if (wallet && !supportsCosmos(wallet)) {
+      const accountSpecifier = `${cosmosChainId}:${ZERO_COSMOS_ADDRESS}`
+      dispatch(getValidatorData.initiate({ accountSpecifier, chainId: cosmosChainId }, options))
+    }
+    if (wallet && !supportsOsmosis(wallet)) {
+      const accountSpecifier = `${osmosisChainId}:${ZERO_OSMOSIS_ADDRESS}`
+      dispatch(getValidatorData.initiate({ accountSpecifier, chainId: osmosisChainId }, options))
```

Comment:
Another way without using those two accounts would be to make `accountSpecifier` optional, since we don't actually need them for the unchained XHR